### DISCUSSION TITLE
Not Pulling User Full Name from messages

### DISCUSSION THREAD

**Discussion by SeLong12** [chatwoot/chatwoot#5971] [Q&A] [answered]
So everything is working with chatwoot in regards to sending and receiving messages via FB Messenger. Only issue is that when someone messages one of our pages, their name is filled with an ALIAS like John Doe, etc. Is there a way to pull the info from messenger to show the full name, this is also preventing me from getting advanced access on FB for user bossiness profile access according to them.

Thanks!

**Comment by pranavrajs** [comment, marked as answer]
@SeLong12 Please apply for production access with your developer app as described here. https://www.chatwoot.com/docs/self-hosted/configuration/features/integrations/facebook-channel-setup/#going-into-production

**Reply by SeLong12** [reply]
So this has been done with the exception that "Business Asset User Profile Access" was denied because Chatwoot is not pulling the users names from messenger. Per Facebook as to why it was denied:

"With regards to your response, your app has to be able to pull the user's name in the app. Thus our reviewer was not able to approve as its showing other names despite being able to receive the message."
